htdocs/imports/import.php

```diff
@@ -1958,7 +1985,10 @@
 
 	//var_dump($array_match_file_to_database);
 
-	$db->begin();
+	$run_after_import = true;
+	if (empty($import_force)) {
+		$db->begin();
```

Comment:
begin and commit/rollback seems not correctly balanced.
If we have this condition on begin, we should also found it for both the rollback and the commit.